example/locals.tf

```diff
@@ -125,13 +125,13 @@ locals {
   token    = "abcdefg" # Dummy #jsondecode(data.aws_secretsmanager_secret_version.terraform_secrets.secret_binary).trend.token
 
   policyid = {
```

Comment:
Would it not be better these as variables to the module instead and then set in internal-compute?

Reply:
They are these are for locat testing